test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -102,9 +104,33 @@ private void Context_Log(object sender, LoggingEventArgs e)
             //throw new NotImplementedException();
         }
 
-        [Fact]
-        public void AutoFitRegressionTest()
+        [Theory]
+        [InlineData("en-US")]
+        [InlineData("ar-SA")]
+        [InlineData("pl-PL")]
+        public void AutoFitRegressionTest(string culture)
         {
+            var originalCulture = Thread.CurrentThread.CurrentCulture;
+            Thread.CurrentThread.CurrentCulture = new CultureInfo(culture);
+
+            uint experimentTime = 0;
+
+            if (culture == "ar-SA")
```

Comment:
I might recommend forming as:
```suggestion
            if (culture != "en-US")
```

Thus giving runtime to each of the less well tested cultures. And then we can remove the `pl-PL` section and use the same runtime (as we need it run the same number of iterations).

Reply:
Yeah, I'm also thinking using the same runtime for pl and ar-SA, but just now I gave them different runtime to try to debug this on the CI and see if the test passes with different runtimes. It so happens that on windows x86 both ar-SA and pl-PL fail with the exception bellow, even if pl-PL is using 100s. Since I haven't been able to reproduce this locally, I haven't been able to debug this.

My first impression was that maybe there's another bug on AutoML that for some reason it might end the experiment abruptly and add a null reference on the `.RunDetails` enumerator, but I'm not yet sure of why would this only happen on Windows x86.

The exception message is:

```
2020-10-29T00:23:05.9676916Z [xUnit.net 00:02:19.90]     Microsoft.ML.AutoML.Test.AutoFitTests.AutoFitRegressionTest(culture: "ar-SA") [FAIL]
2020-10-29T00:23:05.9698138Z   X Microsoft.ML.AutoML.Test.AutoFitTests.AutoFitRegressionTest(culture: "ar-SA") [34s 716ms]
2020-10-29T00:23:05.9698700Z   Error Message:
2020-10-29T00:23:05.9699132Z    System.NullReferenceException : Object reference not set to an instance of an object.
2020-10-29T00:23:05.9701674Z   Stack Trace:
2020-10-29T00:23:05.9702252Z      at Microsoft.ML.AutoML.Test.AutoFitTests.<>c.<AutoFitRegressionTest>b__6_0(RunDetail`1 i) in D:\a\1\s\test\Microsoft.ML.AutoML.Tests\AutoFitTests.cs:line 146
2020-10-29T00:23:05.9703243Z    at System.Linq.Enumerable.Max[TSource,TResult](IEnumerable`1 source, Func`2 selector)
2020-10-29T00:23:05.9711767Z    at Microsoft.ML.AutoML.Test.AutoFitTests.AutoFitRegressionTest(String culture) in D:\a\1\s\test\Microsoft.ML.AutoML.Tests\AutoFitTests.cs:line 146
```

Reply:
Or tersely:
```suggestion
            uint experimentTime = culture == "en-US" ? 0 : 100;
```

Reply:
Thanks.
#Resolved